config/action.d/firewallcmd-common.conf

```diff
@@ -5,72 +5,102 @@
 
 [Init]
 
+# Option:  bantime
+# Notes:   Specifies the bantime in seconds (handled internally rather than by fail2ban)
+# Values:  [ NUM ]  Default: 600
+
+bantime = 600
```

Comment:
BTW... I wonder if ideally we should somehow flag (or interface this bantime into fail 2ban) such actions which time out independently of our internal logic. Since eg of you set bantime here shorter than internally known bantime , wouldn't we even not try to ban until internal timer for the ban runs out, @sebres?